_includes/manuals/1.16/1.2_release_notes.md

```diff
@@ -3,9 +3,16 @@
 ### Headline features
 
 #### Netgroup LDAP authentication
+Netgroup grouping is alternative to posix usergroups. Foreman can now be configured to pull users from external groups using either posix or Netgroup naming standards. Although this is an older standard, it can be useful and is supported for integration with [FreeIPA](https://www.freeipa.org/) and [OpenLDAP](https://www.openldap.org/). ([#16112](http://projects.theforeman.org/issues/16112))
+
 #### Puppet 5 support
+Foreman can be integrated with Puppet 5 puppet masters. Users who are upgrading should reference the [Puppet 4 to Puppet 5 Tutorial](http://projects.theforeman.org/projects/foreman/wiki/Upgrading_from_Puppet_4_to_5) for more information.  ([#15096](http://projects.theforeman.org/issues/15096))
+
 #### ActiveJob + Dynflow backend for background jobs
+ActiveJob is a very simple and well understood means of scheduling background jobs. This initial integration provides a consistent API over the [Dynflow engine](http://dynflow.github.io/). The goal is to encourage more background jobs to encourage more dynamic user workflows. ([#18618](http://projects.theforeman.org/issues/18618))
```

Comment:
as a user I would have probably no idea what this is all about 😖

Reply:
Perhaps saying something like we have background processing available now would help

Reply:
Yeah, this is mostly a feature for plugin developers, probably worth mentioning

Reply:
reworked. lemme know what you think.